sdk/frontdoor/Azure.ResourceManager.FrontDoor/api/Azure.ResourceManager.FrontDoor.netstandard2.0.cs

```diff
@@ -902,21 +975,21 @@ public partial class LatencyMetric
         public float? BValue { get { throw null; } }
         public float? Delta { get { throw null; } }
         public float? DeltaPercent { get { throw null; } }
-        public string EndDateTimeUTC { get { throw null; } }
+        public System.DateTimeOffset? EndDateTimeUtc { get { throw null; } }
```

Comment:
maybe `EndOn`?

Reply:
Renamed, and also renamed `StartDateTimeUtc`